src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Blocks/footer.html.twig

```diff
@@ -106,7 +106,15 @@
       >
         {{ 'Add new product'|trans({}, 'Admin.Catalog.Feature')|raw }}
       </button>
-      {% if isProductPageV2Enabled == true %}
+      {% if isProductPageV2Enabled == true and isCreationMode is same as(true) %}
```

Comment:
```suggestion
      {% if isProductPageV2Enabled == true and isCreationMode == true %}
```

Reply:
Isn't it better to enforce strict equality?

Reply:
You're right, it's quite better to use `is same as` I was thinking `==` is enough but no :sweat: 
But, keep in mind, when using `is same as`, null value will not return false :)